conf/psc/l10n.yml

```diff
@@ -292,6 +293,7 @@ local-italy-maintainers:
     - francesco-ooops
     - SirAionTech
     - MarcoCalcagni
+    - Borruso
```

Comment:
```suggestion
```
Too many times I have asked @Borruso to:

- please squash the administrative commits
- please fix the commit message
- please add a test
- please describe the issue that is being fixed
- please fix the CI

Since the maintainers should enforce the rules, they should be the first ones to follow them.

Moreover, being @Borruso an employee of @MarcoCalcagni I don't see any added value for them in being both maintainers.

Reply:
P.S. Don't PSC members have to sign the CLA?